pkg/scalers/gcp_cloud_tasks_scaler.go

```diff
@@ -179,5 +179,5 @@ func (s *gcpCloudTasksScaler) getMetrics(ctx context.Context, metricType string)
 
 	// Cloud Tasks metrics are collected every 60 seconds so no need to aggregate them.
 	// See: https://cloud.google.com/monitoring/api/metrics_gcp#gcp-cloudtasks
-	return s.client.GetMetrics(ctx, filter, s.metadata.projectID, nil)
+	return s.client.GetMetrics(ctx, filter, s.metadata.projectID, nil, nil)
```

Comment:
Does it make sense adding support to this also in cloud_tasks?

Reply:
Hi,
the metric "cloudtasks.googleapis.com/queue/depth" seems to always have a value set >=0.
So i don't think it make sense.